core/storage/base_model/gae_models.py

```diff
@@ -103,8 +103,18 @@ def get_multi(cls, entity_ids, include_deleted=False):
             not found, or it has been deleted and include_deleted is False,
             then the corresponding entry is None.
         """
-        entity_keys = [ndb.Key(cls, entity_id) for entity_id in entity_ids]
+        entity_keys = []
```

Comment:
This needs comprehensive tests. In particular be careful to test the case of multiple things missing in irregular places, since the correctness is somewhat dependent on the order of insertions.